File: Poin B/PHP/SoccerClub.php
<?php


class SoccerClub{
    //atribut
    private $Name = "";
    private $Nation = "";
    private $Year = "";
    private $Players = array();

    //konstruktor
    function __construct($Namec = "", $Nationc = "", $Yearc = "", $Playersc = "")
    {
        $this->Name = $Namec;
        $this->Nation = $Nationc;
        $this->Year = $Yearc;
        $this->Players = $Playersc;        
    }

    //methods get set
    function setName($Namec){
        $this->Name = $Namec;
    }
    
    function setNation($Nationc){
        $this->Nation = $Nationc;
    }

    function setYear($Yearc){
        $this->Year = $Yearc;
    }

    function setPlayers($Playersc){
        $this->Players = $Playersc;
    }

    function getName(){
        return $this->Name;
    }

    function getNation(){
        return $this->Nation;
    }

    function getYear(){
        return $this->Year;
    }

    function getPlayers(){
        return $this->Players;
    }

    function output(){
        echo "Name Team : ". $this->getName()."<br>";
        echo "Country : ". $this->getNation() ."<br>";
        echo "Established : ". $this->getYear()."<br>";
        echo "Players : <br>";

        for($i = 0; $i < count($this->getPlayers()); $i++){
            echo "&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp- ".$this->getPlayers()[$i]."<br>";
        }
    }


}
?>

File: Poin B/C++/SoccerClub.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

class SoccerClub{
    private:
        //atribut
        string name;
        string nation;
        string year;
        vector<string> players;

    public:
        SoccerClub(){
        }
        SoccerClub(string namec, string nationc, string yearc, vector<string> playersc){
            this->name = namec;
            this->nation = nationc;
            this->year = yearc;
            this->players.insert(this->players.end(), playersc.begin(), playersc.end());
        };

        //get set methods
        void setName(string namec){
            this->name = namec;
        }
        string getName(){
            return this->name;
        }
        void setNation(string nationc){
            this->nation = nationc;
        }
        string getNation(){
            return this->nation;
        }
        void setYear(string yearc){
            this->year = yearc;
        }
        string getYear(){
            return this->year;
        }
        void setPlayers(vector<string> playersc)
        {
            this->players.insert(this->players.end(), playersc.begin(), playersc.end());
        }
        vector<string> getPlayers(){
            return this->players;
        }
        

        void output(){
            cout << "Name Team      : " + this->getName() << endl;
            cout << "Country        : " + this->getNation() << endl;
            cout << "Established    : " + this->getYear() << endl;
            cout << "Players        : " << endl;
            int i = 0;

            for (i = 0; i<this->players.size(); i++){
                cout << "                   - " + this->players[i] << endl;
            }
    	}
    //Destructor
    ~SoccerClub();
};
